packages/tools/kadena-cli/src/devnet/utils/devnetHelpers.ts

```diff
@@ -93,11 +93,11 @@ export function defaultDevnetIsConfigured(): boolean {
 
 export function getDevnetConfiguration(
   name: string,
-): IDevnetsCreateOptions | null {
+): IDevnetsCreateOptions | undefined {
   const devnetFilePath = path.join(defaultDevnetsPath, `${name}.yaml`);
 
   if (!PathExists(devnetFilePath)) {
-    return null;
+    return undefined;
```

Comment:
```suggestion
    return;
```
when you don't return any specific value then by default it's `undefined` .